examples/6tisch/etsi-plugtest-2017/Makefile

```diff
@@ -2,7 +2,7 @@ CONTIKI_PROJECT = node
 all: $(CONTIKI_PROJECT)
 
 PLATFORMS_EXCLUDE = sky nrf52dk native simplelink
-BOARDS_EXCLUDE = srf06/cc13xx launchpad/cc1310 launchpad/cc1350 sensortag/cc2650 sensortag/cc1350
+BOARDS_EXCLUDE = sensortag/cc2650 sensortag/cc1350
```

Comment:
I'm curious, why do these two boards need excluding?

Reply:
Running out of RAM

Reply:
Has this problem of running out of RAMbeen solved? How did you solve this problem?